docs/measurement/schema/response.md

```diff
@@ -1,934 +0,0 @@
-# Measurement Response Schema
-
-Jump to:
-- [`Full success schema`](#success)
-- [`Result schema`](#result)
-- [`Ping Result schema`](#ping)
-- [`Traceroute  Result schema`](#traceroute)
-- [`DNS Result schema`](#dns)
-- [`MTR Result schema`](#mtr)
-- [`HTTP Result schema`](#http)
-
-## Success
-
-### schema
-
-#### Id
-
-**key**: `id`
-
-**type**: `string`
-
-**mandatory**: `true`
-
-Measurement Id number, obtained from [`POST /v1/measurements`](../post-create.md) request.
-
-#### type
-
-**key**: `type`
-
-**type**: `string`
-
-**mandatory**: `true`
-
-The type of measurement. For more details about measurement type, please see [`MEASUREMENT REQUEST SCHEMA`](./request.md)
-
-**available values**:
-- `ping`
-- `dns`
-- `traceroute`
-- `mtr`
-
-#### status
-
-**key**: `status`
-
-**type**: `string`
-
-**mandatory**: `true`
-
-Measurement's current status. Communication between server and probes is asynchronious, thus the response should not be considered final, until this field returns `finished`.
-
-**available values**:
-- `finished`
-- `in-progress`
-
-#### createdAt / updatedAt
-
-**key**: `createdAt` `updatedAt`
-
-**type**: `string` (ISO)
-
-**mandatory**: `true`
-
-#### target
-
-**key**: `target`
-
-**type**: `string`
-
-**mandatory**: `true`
-
-An endpoint on which the tests were executed. For more details about measurement target, please see [`MEASUREMENT REQUEST SCHEMA`](./request.md)
-
-#### limit
-
-**key**: `limit`
-
-**type**: `number`
-
-**mandatory**: `false`
-
-The global limit of probes that was specified in the request body. For more details about measurement limit, please see [`MEASUREMENT REQUEST SCHEMA`](./request.md)
-
-#### probesCount
-
-**key**: `probesCount`
-
-**type**: `number`
-
-**mandatory**: `true`
-
-The number of probes that performed the measurement. `probesCount` is always smaller or equal to `limit`.
```

Comment:
`probesCount` added. I intentionally didn't include descriptions for things that should be obvious because I think writing "id - the measurement id" just adds extra text to read with no value.